clients/client-python/gravitino/dto/dto_converters.py

```diff
@@ -20,14 +20,14 @@ def to_metalake_update_request(change: MetalakeChange) -> object:
         # Assuming MetalakeUpdateRequest has similar nested class structure for requests
         if isinstance(change, MetalakeChange.RenameMetalake):
             return MetalakeUpdateRequest.RenameMetalakeRequest(change.new_name())
-        elif isinstance(change, MetalakeChange.UpdateMetalakeComment):
+        if isinstance(change, MetalakeChange.UpdateMetalakeComment):
             return MetalakeUpdateRequest.UpdateMetalakeCommentRequest(change.new_comment())
-        elif isinstance(change, MetalakeChange.SetProperty):
+        if isinstance(change, MetalakeChange.SetProperty):
             return MetalakeUpdateRequest.SetMetalakePropertyRequest(change.property(), change.value())
-        elif isinstance(change, MetalakeChange.RemoveProperty):
+        if isinstance(change, MetalakeChange.RemoveProperty):
             return MetalakeUpdateRequest.RemoveMetalakePropertyRequest(change.property())
-        else:
-            raise ValueError(f"Unknown change type: {type(change).__name__}")
```

Comment:
Why need change `elif` to `if`?

Reply:
This is suggested by pylint, the no-else-return rule
https://pylint.pycqa.org/en/latest/user_guide/messages/refactor/no-else-return.html